### DISCUSSION TITLE
related data is not in the output

### DISCUSSION THREAD

**Discussion by rabol** [GeorgeFourkas/ApiAutoPilot#13] [Q&A]
Hi

I can see that the AAP tries to find relations, but it never output the related data.

Do I miss a setting ?

**Comment by GeorgeFourkas** [comment]
hi, can you please show what the request url is? and what is the name of the method in your model class that defines your relatioship ?

**Reply by rabol** [reply]
the request looks like this:

https://wifdt.test/api/aap/recipe

```
 public function ingredients(): HasMany
    {
        return $this->hasMany(RecipeIngredient::class);
    }

    public function instructions(): HasMany
    {
        return $this->hasMany(RecipeInstruction::class);
    }
```
